spark/v3.2/spark/src/test/java/org/apache/iceberg/spark/TestSparkDistributionAndOrderingUtil.java

```diff
@@ -1300,6 +1300,202 @@ public void testRangeCopyOnWriteMergePartitionedSortedTable() {
     checkCopyOnWriteDistributionAndOrdering(table, MERGE, expectedDistribution, expectedOrdering);
   }
 
+  // ===================================================================================
+  // Distribution and ordering for merge-on-read DELETE operations with position deletes
+  // ===================================================================================
+  //
+  // UNPARTITIONED
+  // -------------------------------------------------------------------------
+  // delete mode is NOT SET -> ORDER BY _spec_id, _partition, _file, _pos
```

Comment:
This should be fine since there is no partition, but you technically don't need _spec_id and _partition.

Reply:
Why default to RANGE here? Is this what we recommend? I'd probably default to HASH because there will be few files. RANGE could take a single file worth of deletes and spread them across 200 delete files accidentally.

Reply:
Oh, I see. Other specs are why we need to include _spec_id and _partition. Good call.